packages/launchpad/cypress/e2e/integration/global-mode.spec.ts

```diff
@@ -0,0 +1,204 @@
+      cy.get('[data-cy="dropzone"]')
+      .find('button[aria-label="Close"]')
+      .click()
+
+      cy.get('[data-cy="dropzone"]').should('not.exist')
+
+      cy.get('[data-cy="addProjectButton"').click()
+      cy.get('[data-cy="dropzone"]').should('exist')
+
+      cy.get('[data-cy="addProjectButton"').click()
+      cy.get('[data-cy="dropzone"]').should('not.exist')
+    })
+
+    describe('Project card menu', () => {
+      it('can be opened', () => {
+        setupProjects(['todos'])
+        cy.log('Project cards have a menu can be opened')
+        cy.get('[data-cy="project-card"] > button').click()
+
+        cy.get('[data-cy="project-card-menu-items"] > button').then((menu) => {
+          expect(menu).to.have.lengthOf(3)
+          expect(menu.get(0)).to.contain(defaultMessages.globalPage.removeProject)
+          expect(menu.get(1)).to.contain(defaultMessages.globalPage.openInIDE)
+          expect(menu.get(2)).to.contain(defaultMessages.globalPage.openInFinder)
+        })
+      })
+
+      it('removes project from list when clicking "Remove Project" menu item', () => {
+        const projectList = ['todos', 'cookies']
+
+        setupProjects(projectList)
+        cy.get('[data-cy="project-card"] > button').then((menu) => {
+          menu.get(0).click()
+        })
+
+        cy.get('[data-cy="Remove Project"]').click()
+
+        cy.get('[data-cy="project-card"]')
+        .should('have.length', 1)
+        .should('contain', projectList[1])
+      })
+
+      it('shows file drop zone when no more projects are in list when clicking "Remove Project" menu item', () => {
+        setupProjects(['todos'])
+        cy.get('[data-cy="project-card"] > button').click()
+        cy.get('[data-cy="Remove Project"]').click()
+
+        cy.get('[data-cy="project-card"]').should('not.exist')
+        cy.get('[data-cy="dropzone"]')
+        .should('contain', defaultMessages.globalPage.empty.dropText.split('{0}')[0])
+      })
+    })
+
+    describe('Searching the project list', () => {
+      const projectList = ['todos', 'ids', 'cookies', 'plugin-empty']
+
+      it('filters project results when searching by project name', () => {
+        setupProjects(projectList)
+        cy.log('Searching for a project by the project name will filter the project results to only show that project')
+        cy.get('#project-search').type('tod')
+        cy.get('[data-cy="project-card"]')
+        .should('have.length', 1)
+        .contains('todos')
+        .contains(path.join('cy-projects', 'todos'))
+
+        cy.log('Deleting the search pattern restores the project list')
+        cy.get('#project-search').type('{backspace}{backspace}{backspace}')
+        cy.get('[data-cy="project-card"]')
+        .should('have.length', projectList.length)
+      })
+
+      // FIXME: fix Search by project path logic
+      it.skip('filters project results when searching by project path', () => {
```

Comment:
Is this fixable at this point?

Reply:
The Component team needs to make the updates for this TO DO to be finished. This is stubbed for when this is added.

Reply:
I've added this to UNIFY-646.